scvi/external/scbasset/_model.py

```diff
@@ -192,6 +203,193 @@ def get_cell_bias(self) -> np.ndarray:
         """
         return self.module.cell_bias.cpu().numpy()
 
+    @classmethod
+    def _download_motifs(cls, genome: str, motif_dir: str) -> None:
+        """Download a set of motifs injected into peaks"""
+        logger.info(f"Downloading motif set to: {motif_dir}")
+        # download the motif set
+        url_name = cls.MOTIF_URLS.get(genome, None)  # (url, dir_name)
+        if url_name is None:
+            raise ValueError(f"{genome} is not a supported motif set.")
+        urlretrieve(url_name[0], filename=Path(motif_dir, f"{genome}_motifs.tar.gz"))
+        # untar it
+        import tarfile
+
+        def rename_members(tarball):
+            """Rename files in the tarball to remove the top level folder"""
+            for member in tarball.getmembers():
+                if member.path.startswith(url_name[1]):
+                    member.path = member.path.replace(url_name[1] + "/", "")
+                    yield member
+
+        # importing the "tarfile" module
+        tarball = tarfile.open(Path(motif_dir, f"{genome}_motifs.tar.gz"))
+        tarball.extractall(path=motif_dir, members=rename_members(tarball))
+        tarball.close()
+
+        # `motif_dir` now has `shuffled_peaks_motifs` as a subdir and
+        # `shuffled_peaks.fasta` as a root level file.
+        logger.info("Download and extraction complete.")
+        return
+
+    def _get_motif_library(
+        self, tf: str, genome: str = "human", motif_dir: str = None
+    ) -> Tuple[List[str], List[str]]:
+        """Load sequences with a TF motif injected from a pre-computed library
+
+        Parameters
+        ----------
+        tf : str
```

Comment:
no need for types here again

Reply:
it pains me to remove types, but I'll do it for you

Reply:
they will get rendered on readthedocs! Though won't be there in an IDE (but can see the signature on an IDE easily)

Reply:
I jest, should be cleaned up :)